x/incentive/keeper/rewards_supply.go

```diff
@@ -187,8 +187,8 @@ func (k Keeper) SynchronizeHardLiquidityProviderClaim(ctx sdk.Context, owner sdk
 		k.SynchronizeHardBorrowReward(ctx, borrow)
 	}
 
-	// Synchronize any hard delegator rewards
-	k.SynchronizeHardDelegatorRewards(ctx, owner, nil, false)
+	// Synchronize any delegator rewards
```

Comment:
Why is it necessary to sync delegator rewards here?

Reply:
Good catch. Removed in https://github.com/Kava-Labs/kava/pull/948/commits/16e95aa1bcd8f71fc870cc7684de2e30865620f5.